Week3/homework/2-step3.js

```diff
@@ -1,8 +1,22 @@
 'use strict';
+function sayThree(number) {
+  console.log(number + "  Is divisible by three ");
+}
+function sayFive(number) {
+  console.log(number + "  Is divisible by five");
+}
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const values = [];
-  // Add your code here
+  for (let z = startIndex; z <= stopIndex; z++) {
+    values.push(z);
```

Comment:
You are not completely following the assignment: it stated that creation of the array and testing for divisibility by 3 and 5 are to be done in two separate steps:

>The function should first generate an array containing values from start value to end value (inclusive).
>
>Then the function should take the newly created array and iterate over it, and calling the first callback if the array value is divisible by 3.

Apart from that, everything is OK here.